lib/avo/base_resource.rb

```diff
@@ -92,6 +93,7 @@ def find_scope
       end
 
       def authorization
+        # Is there a way to pass the authorization_policy here as well ?
```

Comment:
you have acces to `authorization_policy`, `model_class` and `Avo::App.current_user` here